lib/index.js

```diff
@@ -19,9 +20,10 @@ module.exports = curry['default'] = curry
 function curry (fn) {
   var self = this
   return function () {
-    var args1 = slice(arguments)
+    var args = slice(arguments)
```

Comment:
I think that the proper solution would be to just update @f/slice - so that it's using this loop internally instead of `Array.prototype.slice`, we should probably do the same thing with concat etc.

I'm happy to do that myself.